### ISSUE TITLE
Copy notes without going to edit mode

### ISSUE THREAD

**Issue by Sapkotaanish** [clipto-pro/Desktop#127]
Is there a way to select and copy the partial content of note without actually going in edit mode.
Like I want to copy some part of text inside the red line:

![Screenshot from 2021-12-29 13-06-56](https://user-images.githubusercontent.com/38289476/147646427-3aa821d8-3db2-4338-abcf-2c5525b7e91f.png)

**Comment by atrashler** [comment]
> Is there a way to select and copy the partial content of note without actually going in edit mode. Like I want to copy some part of text inside the red line:
> 
> ![Screenshot from 2021-12-29 13-06-56](https://user-images.githubusercontent.com/38289476/147646427-3aa821d8-3db2-4338-abcf-2c5525b7e91f.png)

1) The standard selection and 'Copy' action in the contextual menu
2) The standard selection and 'Ctrl+C' shortcut

Probably the standard selection does not work on your platform, does it?

**Comment by Sapkotaanish** [comment]
No, standard selection doesn't work in my platform. I am using Ubuntu20.04
